google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/features/InstanceApi.java

```diff
@@ -236,6 +236,7 @@ Operation setDiskAutoDelete(@PathParam("instance") String instanceName,
    /**
     * Sets an instance's scheduling options.
     * @see <a href = "https://cloud.google.com/compute/docs/instances#onhostmaintenance"/>
+    * @see #setScheduling(String, org.jclouds.googlecomputeengine.domain.Instance.Scheduling.OnHostMaintenance, boolean, boolean)
```

Comment:
@nacx Do we do this kind of JavaDoc referencing to other overloaded versions? Or drop it?